src/vs/editor/contrib/codelens/browser/codelensWidget.ts

```diff
@@ -98,7 +98,7 @@ class CodeLensContentWidget implements IContentWidget {
 			if (lens.command) {
 				const title = renderLabelWithIcons(lens.command.title.trim());
 				if (lens.command.id) {
-					children.push(dom.$('a', { id: String(i), title: lens.command.tooltip }, ...title));
+					children.push(dom.$('a', { id: String(i), title: lens.command.tooltip, role: 'button' }, ...title));
```

Comment:
This change makes sense to me.
fyi @jrieken
This fixes https://github.com/microsoft/vscode/issues/145774